src/main.rs

```diff
@@ -320,6 +329,13 @@ fn watch(initial_path: Option<&Path>, format: bool, args: Vec<String>) -> io::Re
             .last()
         {
             if last_time.elapsed() > Duration::from_millis(100) {
+                if clear {
+                    if cfg!(target_os = "windows") {
+                        Command::new("cmd").args(&["/C", "cls"]).status().unwrap();
```

Comment:
`_ = ` to discard the error is probably better here.
You never know what might be available or not

Reply:
Changed